projects/jsf-launcher/src/app/schemaV2.json

```diff
@@ -160,14 +160,31 @@
         "dropdowns": {
           "type": "object",
           "name": "Dropdowns",
-          "required": ["dropdown1"],
-          "description": "Dropdowns can be required or not required. If no values are passed in, the form will display \"Select an option...\" by default. If the dropdown is required, then the default option will be hidden once the user has selected a value. If the dropdown is not required, the default option will always be present.",
+          "description": "A value must be selected from a dropdown. A default enum value may be specified",
           "properties": {
             "dropdown1": {
               "display": "dropdown",
               "name": "Dropdown",
               "helpText": "This help text will display when you hover over the information icon",
               "tooltip": "This tooltip will display when you hover over the label.",
+              "enum": [
+                "",
+                "option1",
+                "option2",
+                "option3"
+              ],
+              "enumNames": [
+                "None",
+                "Option 1",
+                "Option 2",
+                "Option 3"
+              ]
+            },
+            "dropdown2": {
+              "display": "dropdown",
```

Comment:
We have a matching data file for this schema called `dataV2.json`. This represents the data that is now being passed back into the JSF when the user comes back to edit the form. Since we're adding new required fields, the `dataV2.json` should also be updated so that it is in sync with the schema.